<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterSettingDialog</class>
 <widget class="QDialog" name="CharacterSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>类型：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeComboBox">
       <item>
        <property name="text">
         <string>NPC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Enemy</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="HPMaxLabel">
       <property name="text">
        <string>MaxHP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="MaxHPSpinBox">
       <property name="maximum">
        <number>9999999</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="MPMaxLabel">
       <property name="text">
        <string>MaxMP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="MaxMPSpinBox">
       <property name="maximum">
        <number>9999999</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="HPLabel">
       <property name="text">
        <string>HP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="HPSpinBox">
       <property name="maximum">
        <number>9999999</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="MPLabel">
       <property name="text">
        <string>MP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="MPSpinBox">
       <property name="maximum">
        <number>99999999</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="ZLabel">
       <property name="text">
        <string>Z Value:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ZSpinBox">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CharacterSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CharacterSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
